pytest_plugins/requirements/req_updater.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+from pathlib import Path
+from functools import cached_property
+
+class ReqUpdater:
+
+    # Installed package name as key and its counterpart in requirements file as value
+    package_deviates = {'Betelgeuse': 'betelgeuse', 'broker': 'broker[docker]', 'dynaconf': 'dynaconf[vault]'}
+
+    @cached_property
+    def installed_packages(self):
+        """Returns the list of installed packages in venv
+
+        This also normalizes any package names that deviates in requirents file vs installed names
+        """
+        installed_pkges = str(subprocess.run('pip list --format=freeze', capture_output=True, shell=True).stdout.decode())
```

Comment:
```suggestion
        installed_pkges = subprocess.run('pip list --format=freeze', capture_output=True, shell=True).stdout.decode()
```
The decode method returns the str object from bytes; there is no need to convert str again.